src/js/formats/expandable.js

```diff
@@ -18,22 +18,39 @@ export default {
         };
     },
     postRender() {
-        q('.js-expand').forEach(el => el.addEventListener('click', () => {
-            q('.js-short-content').forEach(shortContentElement => {
-                shortContentElement.classList.add('u-hidden');
-            });
-            q('.js-all-content').forEach(allContentElement => {
-                allContentElement.classList.remove('u-hidden');
-            });
-        }));
-        q('.js-collapse').forEach(el => el.addEventListener('click', () => {
-            q('.js-all-content').forEach(allContentElement => {
-                allContentElement.classList.add('u-hidden');
+        const $brexit = q('.brexit')[0];
+        const $expandLink = q('.js-expand')[0];
+        const $collapseLink = q('.js-collapse')[0];
+        const $shortContent = q('.js-short-content')[0];
+        const initialHeight = $brexit.offsetHeight;
+
+        function expand() {
+            q('.js-all-content').forEach((el) => {
+                el.classList.remove('u-hidden');
             });
-            q('.js-short-content').forEach(shortContentElement => {
-                shortContentElement.classList.remove('u-hidden');
+            $shortContent.classList.add('u-hidden');
+
+            // set to an excessive max height
+            $brexit.style.maxHeight = '1000px';
+
+            // then restore to a reasonable max height after transition
+            setTimeout(() => {
+                $brexit.style.maxHeight = `${$brexit.offsetHeight}px`;
+            }, 300);
```

Comment:
Could you do this with a `transitionend` listener rather than having the value `300` tied to a value in the CSS?